x-pack/plugins/uptime/server/lib/requests/get_network_events.ts

```diff
@@ -20,7 +20,12 @@ export const secondsToMillis = (seconds: number) =>
 
 export const getNetworkEvents: UMElasticsearchQueryFn<
   GetNetworkEventsParams,
-  { events: NetworkEvent[]; total: number; isWaterfallSupported: boolean }
+  {
```

Comment:
It could be nice to either create a named interface or runtime type for re-use on the client. It's helpful for tracing to have the interface referenced in both places, so someone researching a problem on the client can quickly find the shared references on the server.